spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -311,6 +313,7 @@ function chanOpenTry(
   connectionHops: [Identifier],
   portIdentifier: Identifier,
   channelIdentifier: Identifier,
+  provedIdentifier: Identifier,
```

Comment:
Is this proved or provided?

Reply:
It's the identifier which was proved on the counterparty side